src/app/recents/recents.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+import { TabsModule } from 'ngx-bootstrap/tabs';
+
+import { NodeTypeDisplayPipe } from '../pipes/nodedisplay';
+import { RecentsService } from '../services/recents.service';
+
+
+@Component({
+  selector: 'app-recents',
+  templateUrl: './recents.component.html',
+  styleUrls: ['./recents.component.css']
+})
+export class RecentsComponent implements OnInit {
+
+  recentsAll: Array<any>; // all recents of every type
+  meta: Array<any>; // metadata
+  types: Array<string>; // types of artifacts
+  loading: boolean;
+  tableSpacing: false;
+  csvName = 'Recent Artifacts';
+
+  constructor(private recentsService: RecentsService) {
+    this.loading = true;
+    this.types = [];
+  }
+
+  ngOnInit() {
+    this.getRecentsAll();
+  }
+
+  getRecentsAll() {
+    // Gets recent artifacts of all types and populates recentsAll
+    this.recentsService.getRecents().subscribe(
+      recentsAll => {
+        this.recentsAll = recentsAll['data'];
+        this.meta = recentsAll['metadata'];
+        this.types = Object.keys(this.recentsAll);
+        this.loading = false;
```

Comment:
I believe this line can be removed because the method that runs when the observable completes already calls this.

Reply:
@mprahl for whatever reason it gets stuck in loading when I remove this line

Reply:
Ah, it's because we are using positional arguments, so the second function is being used as the error handler. You can try passing null as the second argument, or pass an object in. See the following for more details:
https://angular.io/guide/observables#subscribing

I think this mistake is done elsewhere in Estuary. It'd be nice if you fixed that in another PR.  :)